srcpkgs/pdfarranger/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'pdfarranger'
+pkgname=pdfarranger
+version=1.7.1
+revision=1
+build_style=python3-module
+hostmakedepends="
+ ghostscript intltool python3-setuptools python3-distutils-extra python3-pikepdf
+ python3-pip python3-xlib python3-wheel"
```

Comment:
does it really need pip?

Reply:
seems unnecessary. previously also worked without python3-pip (local compile). it's just that i see it in debian dependencies.